src/components/SignInModal.js

```diff
@@ -0,0 +1,46 @@
+import { CancelSignInIcon } from '../icons'
+import React from 'react'
+import '../index.css'
+
+
+const SignInModal = ({setShowSignInModal, setShowSignUpModal}) => {
```

Comment:
We actually don't need authentication for this particular project.. except you just want to add it which is fine

Reply:
Yes, just wanted to add it